src/CommunityToolkit.Maui.Markup.SourceGenerators/Extensions/EquatableArray{T}.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Runtime.CompilerServices;
+
+// Source: https://github.com/CommunityToolkit/dotnet/blob/main/src/CommunityToolkit.Mvvm.SourceGenerators/Helpers/EquatableArray%7BT%7D.cs
+
+namespace CommunityToolkit.Maui.Markup.SourceGenerators;
+
+/// <summary>
+/// Extensions for <see cref="EquatableArray{T}"/>.
+/// </summary>
+static class EquatableArray
```

Comment:
I'm confused as to why were the accessibility modifiers removed..??

Reply:
@Sergio0694 That is the code-style enforced in this repo 😕 

https://github.com/CommunityToolkit/Maui.Markup/blob/ebc006f978f000902b271b39aa29a9a81f480be8/.editorconfig#L50

Reply:
Uuuuhhhhh.... I mean, I have so many questions but ok then 😆